cluster/gce/gci/configure-helper.sh

```diff
@@ -492,16 +492,6 @@ function create-node-pki {
     KUBELET_KEY_PATH="${pki_dir}/kubelet.key"
     write-pki-data "${KUBELET_KEY}" "${KUBELET_KEY_PATH}"
   fi
-
-  if [[ "${ENABLE_EGRESS_VIA_KONNECTIVITY_SERVICE:-false}" == "true" ]]; then
-    mkdir -p "${pki_dir}/konnectivity-agent"
-    KONNECTIVITY_AGENT_CA_CERT_PATH="${pki_dir}/konnectivity-agent/ca.crt"
```

Comment:
If your cleaning this up you should probably also clean up the section at https://github.com/kubernetes/kubernetes/blob/4ae223fbff3dc0fa3a569ca579a147a3bc0f6f8a/cluster/gce/util.sh#L1096

Reply:
Done.